source/common/network/happy_eyeballs_connection_impl.cc

```diff
@@ -66,49 +67,58 @@ bool hasMatchingIpVersion(const Address::IpVersion& ip_version,
 } // namespace
 
 std::vector<Address::InstanceConstSharedPtr> HappyEyeballsConnectionProvider::sortAddresses(
```

Comment:
Following the internal discussion, consider adding a TODO to call this sort function once during the ingestion of the config.

Reply:
I created this OSS issue to track this cleanup: https://github.com/envoyproxy/envoy/issues/32916